src/main/java/com/training/springpractice/controllers/MemberController.java

```diff
@@ -0,0 +1,24 @@
+package com.training.springpractice.controllers;
+
+import com.training.springpractice.models.Member;
+import com.training.springpractice.repositories.MemberRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/v1/members")
+public class MemberController {
+
+    @Autowired
+    private MemberRepository repository;
+
+    @PostMapping
```

Comment:
I saw in other projects that with the mapping GET/POST/PUT etc, is defined the **_produces_** property, Don't we need to define that we are going to return a JSON format?